test/WebJobs.Extensions.Tests/app.config

```diff
@@ -2,6 +2,7 @@
 <configuration>
   <appSettings>
     <add key="NotificationHubsConnectionString" value="ConnectionString_FromAppSettings" />
+    <add key="CosmosDBConnectionString" value="AccountEndpoint=https://someEndpoint;AccountKey=someKey;" />
```

Comment:
What if I wanted to test that the extraction of the value from the AppSettings worked specifically? Not just the defaulting of the environment variable.